src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -9500,6 +9501,31 @@ public class AA
             End Function
         End Class
 
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.Completion)>
+        <WorkItem(63922, "https://github.com/dotnet/roslyn/issues/63922")>
+        Public Async Function DoNotSelectItemWithHigherMatchPriorityButWorseCaseSensitivity() As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+            <Document>
+using System;
+public class AA
+{
+    public void F(object dr)
+    {
+        var DR = (string)dr$$
+    }
+}</Document>)
+
+                state.SendInvokeCompletionList()
+                Await state.WaitForAsynchronousOperationsAsync()
+                Await state.WaitForUIRenderedAsync()
+
+                Await state.AssertSelectedCompletionItem("dr")
+                Await state.AssertCompletionItemsContain("DR", "")
```

Comment:
can we have some tests where it's a prefix match, not just a full match.

Reply:
Sure! I will add one in a follow up PR.